modules/KalturaSupport/components/sideBarContainer.js

```diff
@@ -73,7 +73,14 @@
 					_this.getConfig("minDisplayWidth") <= _this.getPlayer().getWidth() &&
 					_this.getConfig("minDisplayHeight") <= _this.getPlayer().getHeight())){
 					_this.render = true;
-					_this.getComponent().css('height', _this.getPlayer().getVideoHolder().height());
+					//Sidebar height is player height without the top and bottom bars
+					var height = _this.getPlayer().getHeight() -
+						_this.getPlayer().getControlBarContainer().height() -
+						_this.getPlayer().getTopBarContainer().height();
+					_this.getComponent().css('height', height);
+					//If topbar exist then add top value
+					_this.getComponent().css('top', _this.getPlayer().getTopBarContainer().height());
```

Comment:
Actually no - jquery will return an empty jquery elements wrapper array that has the height function - but I agree that it's better practice to include a check here.